src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,26 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
 
+    public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = (ManufacturerDao) injector
+                         .getInstance(ManufacturerDao.class);
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setName("Mercedes");
+        manufacturer.setCountry("Germany");
+        manufacturerDao.create(manufacturer);
+        System.out.println(manufacturerDao.getAll());
+        System.out.println(manufacturerDao.get(4L));
```

Comment:
Do not use a magic ID
```suggestion
        System.out.println(manufacturerDao.get(mercedes.getId()));
```